packages/fabric8-ui/src/app/dashboard-widgets/space-settings-widget/space-settings-widget.component.html

```diff
@@ -0,0 +1,130 @@
+    <div class="card-pf-body f8-small-card-body">
+      <div class="">
+        <p>{{ UserEmail }} ({{ UserName }})</p>
+        <ul class="f8-space-settings-data-links">
+          <li>
+            <span class="pficon pficon-info pficon-users fa-sm"> </span>
+            {{ collaboratorCount }} Collaborators |
+            <a class="add-collaborators-link" (click)="launchAddCollaborators()">
+              <span class="pficon pficon-add-circle-o fa-lg"></span> Add Collaborators
+            </a>
+          </li>
+          <li></li>
+          <li><span class="fa fa-life-saver fa-sm"></span> {{ areaCount }} Areas</li>
+        </ul>
+      </div>
+    </div>
+    <div class="card-pf-footer space-description-footer">
+      <div class=" f8-space-masthead-description">
+        <!-- There must be no spaces here, otherwise we end up with random whitespace in the editable-->
+        <div *ngIf="userOwnsSpace; else userDoesNotOwnSpaceDescription" class="form-group">
+          <textarea
+            row="1"
+            placeholder="Description goes here"
+            class="space-description form-control"
+            *ngIf="isEditing && isEditable()"
+            (keyup.enter)="onUpdateDescription(description.value)"
+            (keydown.enter)="preventDef($event)"
+            #spaceDescription
+            >{{ space?.attributes.description }}</textarea
+          >
+          <div class="f8-space-masthead-description-edit-buttons" *ngIf="isEditing">
+            <button
+              id="_btn_cancel_description"
+              class="pull-right btn btn-default edit-space-description-btn"
+              (click)="stopEditingDescription()"
+            >
+              Cancel
+            </button>
+            <!--Save icon-->
+            <button
+              id="workItemTitle_btn_save_description"
+              class="pull-right btn btn-primary edit-space-description-btn "
+              (click)="saveDescription()"
+            >
+              Save
+            </button>
+          </div>
+          <p
+            *ngIf="!isEditing || !isEditable()"
+            (click)="startEditingDescription()"
+            class="f8-space-masthead-description-body edit-space-description-pencil truncate-multiline"
+          >
+            <span *ngIf="SpaceDescription.length; else enterDescription">
+              {{ SpaceDescription }}
+            </span>
+            <ng-template #enterDescription>
+              <span style="font-style: italic;color:grey">Enter a description</span>
+            </ng-template>
+          </p>
+        </div>
+        <ng-template #userDoesNotOwnSpaceDescription>
+          <p class="truncate-multiline">
+            <span *ngIf="SpaceDescription.length; else noDescription">
+              {{ SpaceDescription }}
+            </span>
+            <ng-template #noDescription>
+              <span style="font-style: italic;color:grey">No description available</span>
```

Comment:
These styles should be in the style file.

Reply:
moved it to the less file